src/main/java/org/mifos/connector/bulk/schema/CsvSchema.java

```diff
@@ -0,0 +1,9 @@
+package org.mifos.connector.bulk.schema;
```

Comment:
duplicate?

Reply:
I have considered 3 options here.

- Add the common code to connector common (I think too many PRs here are causing release management issues at the moment)
- Adding the common code in a new service called bulk common
- Prioritizing the merging of this PR and keeping the duplicate code for some time. I am adding TODO comments in all the duplicate files for the PR to get merged. Creating a new ticket for it which we can take up at a later stage and do the 2nd option.

I believe we should stick to the 3rd option since I don't intend to add more complexity and time to get this PR merged. Do let me know if you are fine with this approach.